region/maridia/inner-pink/Halfie Climb Room.json

```
@@ -856,6 +884,7 @@
           "canSpaceJumpWaterEscape"
         ]}
       ],
+      "flashSuitChecked": true,
       "devNote": "FIXME: With a single underwaterwalljump you can break free here, without spacejump."
```

```suggestion
      "flashSuitChecked": true
```
The `canUnderwaterWalljumpBreakFree` strat could be grouped closer too